admin/DDPCICDRole.yaml

```diff
@@ -0,0 +1,27 @@
+description: 'Enable deployments of Angular apps on GAE'
+etag: BwWgSLxPGqs=
+includedPermissions:
+- appengine.applications.get
+- appengine.applications.update
+- appengine.operations.get
+- appengine.operations.list
+- appengine.services.get
+- appengine.services.list
+- appengine.services.update
+- appengine.versions.create
+- appengine.versions.delete
+- appengine.versions.get
+- appengine.versions.list
+- appengine.versions.update
+- cloudbuild.builds.create
+- cloudbuild.builds.get
+- cloudbuild.builds.list
+- cloudbuild.builds.update
+- storage.objects.create
```

Comment:
Can we remove the `storage.*` items here?  We ended up taking care of this with specific permissions for the prod bucket.

Reply:
I'll have to test it out. I think we needed some of those just for deployment.

Reply:
We need them!